pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: sass
-version: 1.78.0
+version: 1.78.1
```

Comment:
Since nothing has actually been release, we could remove the git tag, and then retag the same release.

Reply:
I agree